Makefile

```diff
@@ -49,6 +49,8 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+NODE_FULL_PATH = $(PWD)/$(NODE_EXE)
```

Comment:
I think @refack mentioned he sets `NODE` before running make sometimes, so I am not sure about replacing the whole thing with a absolute path. I would prefer just replacing `NODE` as well. Also does this get set in the CI anywhere?

Reply:
>I think @refack mentioned he sets NODE before running make sometimes, so I am not sure about replacing the whole thing with a absolute path.

I'm pretty sure `?=` means it will use the `NODE` passed in if set.